lib/Doctrine/Common/Annotations/Annotation/Target.php

```diff
@@ -72,7 +72,7 @@ final class Target
     public $literal;
 
     /**
-     * @param array $values
+     * @param array{value?: string|list<string>} $values
```

Comment:
I tried advanced syntax here but then got a warning that the line was too long and couldn't find a multiline syntax that would not make phpcs crash with `An error occurred during processing; checking has been aborted. The error message was: Uninitialized string offset: 0 in
    |       |     /home/greg/dev/doctrine-annotations/vendor/squizlabs/php_codesniffer/src/Standards/Squiz/Sniffs/Commenting/FunctionCommentSniff.php on line 381`
or complain that `Missing parameter name (Squiz.Commenting.FunctionComment.MissingParamName)`